src/main/scala/nl/knaw/dans/easy/multideposit/actions/AddDatasetMetadataToDeposit.scala

```diff
@@ -168,7 +168,8 @@ object AddDatasetMetadataToDeposit {
         case (None,    Some(l),_      ) => elem(s"dc:relation")(l)
         case (None,    None,   Some(t)) => elem(s"dc:relation")(t)
         case _                          =>
-          // TODO this case needs to be checked to not occur in the preconditions (see also line 32)
+          // TODO this case needs to be checked to not occur in the preconditions
+          // (see also comment in https://github.com/DANS-KNAW/easy-split-multi-deposit/blob/master/src/main/scala/nl/knaw/dans/easy/multideposit/actions/AddDatasetMetadataToDeposit.scala#L32)
```

Comment:
Not sure but I guess github preserves pull request, even if you drop the branch, take a look at previously squash-merged pull request. Anyhow, linking to the master branch just makes the comment longer and doesn't add any persistence.